src/envoy/utils/utils.cc

```diff
@@ -36,21 +37,21 @@ const std::string kMetadataDestinationUID("uid");
 
 }  // namespace
 
-void ExtractHeaders(const Http::HeaderMap& header_map,
-                    const std::set<std::string>& exclusives,
-                    std::map<std::string, std::string>& headers) {
+void ExtractHeaders(const Http::HeaderMap &header_map,
```

Comment:
Could you check when the "&" "*" format is introduced? I don't see this pattern else where

Reply:
I doubt this is caused by the different version of `clang-format` I used.

Reply:
If it is introduced at your merge, you'd better fixed with this PR.
If it is introduced by existing PR, we could fix it first and merge to master